creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -343,10 +396,10 @@ spec:
 At this time, `slirp` mode doesn’t support additional configuration
 fields.
 
-_______________________________________________________________________________________________
-*Note:* in `slirp` mode, the only supported protocols are TCP and UDP.
-ICMP is _not_ supported.
-_______________________________________________________________________________________________
+______________________________________________________________________________
+*Note:* in `slirp` mode, the only supported protocols are TCP and UDP.  ICMP is
```

Comment:
Noooo, two spaces before ICMP

Reply:
Explain why it's bad.

Reply:
It's not consistent across the whole document. If you explain to me a double-space rule that would apply to the whole document, I'm willing to stop nit-picking.

Reply:
It's just what vim's re-wrapping to X chars does this by default. Believe me I haven't put these spaces in manually.

I will remove them but I find it's weird that they get nit-picked while we tend to ignore commit messages, comments, wrapping, copy paste in other places. ;)

Reply:
Never surrender! It's simple, it is easy to point out and suggest solution for double space, not so for a commit message (hidden by evil GitHub). But you are right, we should step up the nit-picking game.

Reply:
Removed two spaces. The Earth is saved!